troposphere/static/js/components/common/tags/Tag.react.js

```diff
@@ -25,10 +25,14 @@ export default React.createClass({
     onClick(e) {
         e.stopPropagation();
         e.preventDefault();
-        RouterInstance.getInstance()
-            .transitionTo("search",null,{ 
-                    q: this.props.tag.get('name')
-            });
+        // Still causing a warning
+        // "Failed Context Types: Required context `router`"
```

Comment:
✔️ OK, this is fixed. When Tag is rendered by a modal we lose the context that Router sets on the parent because our modals are rendered outside of the parent. 

To fix I pulled the route logic out of the tag and use a callback for the event handler.